content/20.results.md

```diff
@@ -68,19 +66,17 @@ Importantly, this model did not achieve as high an accuracy as the model trained
 ### Assessing the Nationality Diversity of Authors and Honorees
 
 We applied our Wiki2019-LSTM model to both our computational biology honorees dataset and our dataset of corresponding authors.
-We found that the proportion of authors in the CelticEnglish categories had decreased (Fig. {@fig:region_breakdown}A), particularly for papers published in _Bioinformatics_ and _BMC Bioinformatics_ (Fig. {@fig:region_breakdown}B).
-Among keynote speakers and fellows we found that the majority of honorees are predicted to be from CelticEnglish countries ({@fig:region_breakdown}C).
-Though sample sizes were small, we did observe some differences between meetings.
-ISMB keynotes had more probability attributable to Israel, while RECOMB had more attributable to EastAsian countries ({@fig:region_breakdown}D).
-When we directly compared honoree composition with PubMed, we observed discrepancies between the two groups, namely a large overrepresentation of CelticEnglish (including American) keynote speakers and a substantial underrepresentation of East Asian keynote speakers ({@fig:region_breakdown}E).
-The proportion of keynote speakers of African, Muslim, and Hispanic origin were also slightly smaller than the field of computational biology as a whole, as represented by corresponding authors in major journals.
+We found that the proportion of authors in the CelticEnglish categories had decreased (Fig. {@fig:region_breakdown}A, left), particularly for papers published in _Bioinformatics_ and _BMC Bioinformatics_ (Supplementary Fig. [S4]A).
+Among keynote speakers and fellows we found that the majority of honorees are predicted to be from CelticEnglish countries ({@fig:region_breakdown}A, right).
+Though sample sizes were small, we did observe some differences in the composition of minority groups between meetings.
+ISMB keynotes had more probability attributable to Israel, while RECOMB had more attributable to EastAsian countries (Supplementary Fig. [S4]B).
+When we directly compared honoree composition with PubMed, we observed discrepancies between the two groups, namely a large overrepresentation of CelticEnglish (including American) keynote speakers and a substantial underrepresentation of East Asian keynote speakers ({@fig:region_breakdown}B).
```

Comment:
There is a mismatch here between CelticEnglish and East Asian. I don't feel strongly on either merged or separate but we should be consistent throughout this section.

Reply:
[Done](https://github.com/greenelab/iscb-diversity-manuscript/pull/18/commits/2ab1dbe519649f8150f4d06d4ce432c5e9a7e035).